code/modules/client/client_procs.dm

```diff
@@ -122,11 +122,12 @@
 
 
 	//Logs all hrefs
-	if(config && config.log_hrefs)
+	if(GLOB.configuration.logging.href_logging)
 		log_href("[src] (usr:[usr]\[[COORD(usr)]\]) : [hsrc ? "[hsrc] " : ""][href]")
 
 	if(href_list["karmashop"])
-		if(config.disable_karma)
+		if(!GLOB.configuration.general.enable_karma)
+			to_chat(src, "Karma is disabled on this server.")
```

Comment:
I think this should be `to_chat(usr, ...)`

Reply:
Both `src` and `usr` are valid in this context since `src` is a client